README.md

```diff
@@ -60,6 +60,7 @@ const saml = new SAML(options);
 - `allowCreate`: grants permission to the identity provider to create a new subject identifier (default: `true`)
 - `spNameQualifier`: optionally specifies that the assertion subject's identifier be returned (or created) in the namespace of another service provider, or in the namespace of an affiliation of service providers
 - `wantAssertionsSigned`: if truthy, add `WantAssertionsSigned="true"` to the metadata, to specify that the IdP should always sign the assertions.
+- `authnRequestsSigned`: if defined, add `AuthnRequestsSigned="false"` to the metadata, to specify that the request is not signed. This option is canceled if you sign your requests.
```

Comment:
Do I understand correctly that if this value is set to `true` or `false`, then `AuthnRequestsSigned="false"` will be added to the metadata? Why use a boolean that way? Also, what do you mean by an option being canceled?

Reply:
The option will be overrided here is it's a signed request.
[https://github.com/node-saml/node-saml/blob/a7de497ba8008cf88f1bc8d5c212e5d268254be3/src/metadata.ts#L69](https://github.com/node-saml/node-saml/blob/a7de497ba8008cf88f1bc8d5c212e5d268254be3/src/metadata.ts#L69)

This options is used in a "logical" way

If a use it a a boolean two use case can appear:
- putting `false` while request is signed ? -> it will be overrided
- putting `true` if the request is not signed -> it is not correct